base/number.jl

```diff
@@ -112,10 +112,36 @@ copysign(x::Real, y::Real) = ifelse(signbit(x)!=signbit(y), -x, +x)
 conj(x::Real) = x
 transpose(x::Number) = x
 ctranspose(x::Number) = conj(x)
-inv(x::Number) = one(x)/x
 angle(z::Real) = atan2(zero(z), z)
 
 """
+    inv(x)
+
+Return the multiplicative inverse of `x`, such that `x*inv(x)` or `inv(x)*x`
+yields [`one(x)`](@ref) (the multiplicative identity).
```

Comment:
Not sure on the best way to word this but probably best to write `essentially yields` instead of `yields`  or something alike that doesn't imply  `inv(x)*x == one(x)`